src/ResourceManagement/HDInsight/HDInsight/Microsoft.Azure.Management.HDInsight.nuget.proj

```diff
@@ -2,8 +2,8 @@
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <ItemGroup>
     <SdkNuGetPackage Include="Microsoft.Azure.Management.HDInsight">
-      <PackageVersion>1.3.0</PackageVersion>
+      <PackageVersion>2.0.0</PackageVersion>
```

Comment:
Then you won't be able to use this library in powershell without fixing these breaks.  Generally, obsoleting a property issues a warning when the property is used. This can become an error when compiled in projects that have 'TreatWarningsAsErrors, however, it doesn't break usage at truntime unless you have added an additional exception